autoload/utils/exec/job.vim

```diff
@@ -21,21 +21,30 @@ function! s:closeBuffer() abort
 endfunction
 
 function! s:createQuickFix() abort
+    " just to be sure all messages were processed
+    sleep 100m
     let l:bufnr = bufnr(s:cmake4vim_buf)
     if l:bufnr == -1
         return
     endif
     let l:old_error = &errorformat
-    if s:err_fmt !=# ''
-        let &errorformat = s:err_fmt
+    if s:cmake4vim_job['err_fmt'] !=# ''
+        let &errorformat = s:cmake4vim_job['err_fmt']
     endif
+
     silent execute 'cgetbuffer ' . l:bufnr
     silent call setqflist( [], 'a', { 'title' : s:cmake4vim_job[ 'cmd' ] } )
-    if s:err_fmt !=# ''
+    if s:cmake4vim_job['err_fmt'] !=# ''
         let &errorformat = l:old_error
     endif
     " Remove cmake4vim job
     let s:cmake4vim_job = {}
+    call s:closeBuffer()
+    if len(s:cmake4vim_jobs_pool) > 0
```

Comment:
What happens if the first job fails? The next one should not run, but currently it does (tried calling `:CMakeRun` for something that fails to compile)

Reply:
Yes maybe you are right.